docker-compose.yml

```diff
@@ -243,7 +243,7 @@ services:
       timeout: 10s
       retries: 30
   geoipupdate:
-    image: "maxmindinc/geoipupdate:v4.7.1"
+    image: "maxmindinc/geoipupdate:v5.1.1"
```

Comment:
[Looks like](https://hub.docker.com/r/maxmindinc/geoipupdate) they are moving to [GHCR](https://github.com/maxmind/geoipupdate/pkgs/container/geoipupdate). Should we update here?

Reply:
It will cause setup problems as the ghcr images require github ghcr docker login to pull

Reply:
Hrm, okay. Looks like 5.1.1 is on both so this is fine for now.